packages/editor/src/store/effects/posts.js

```diff
@@ -256,7 +257,27 @@ export const requestPostUpdateFailure = ( action, store ) => {
 	const noticeMessage = ! isPublished && publishStatus.indexOf( edits.status ) !== -1 ?
 		messages[ edits.status ] :
 		__( 'Updating failed' );
-	dispatch( createErrorNotice( noticeMessage, { id: SAVE_POST_NOTICE_ID } ) );
+
+	const cloudflareDetailsLink = addQueryArgs(
+		'post.php',
+		{
+			post: post.id,
+			action: 'edit',
+			'classic-editor': '',
+			'cloudflare-error': '',
+		} );
+
+	const cloudflaredMessage = error && 'cloudflare_error' === error.code ?
```

Comment:
Typo: `cloudflared` => `cloudflare`

Edit: Or... intentional? I guess?

Reply:
Intentional, the reason for this even needing to exist is silly, and given that it'll (hopefully) be reverted next week, we can be a little silly with the variable names, too.